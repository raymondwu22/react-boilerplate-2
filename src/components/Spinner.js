import React from 'react';

const spinner = () => (
  <div className="loader">
    <div className="loader--image">Loading...</div>
  </div>
);

export default spinner;
