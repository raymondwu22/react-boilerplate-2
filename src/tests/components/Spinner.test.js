import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../../components/Spinner';

test('should render component Spinner', () => {
  const wrapper = shallow(<Spinner />);

  expect(wrapper).toMatchSnapshot();
});
