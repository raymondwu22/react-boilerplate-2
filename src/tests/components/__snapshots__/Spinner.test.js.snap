// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component Spinner 1`] = `
<div
  className="loader"
>
  <div
    className="loader--image"
  >
    Loading...
  </div>
</div>
`;
